uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/sdk/AuthResult.java

```diff
@@ -75,4 +75,8 @@ public void assertFailure() {
         Logger.i(TAG, "Assert Failure if there is exception being set..");
         Assert.assertNotNull(exception);
     }
+
+    public boolean isAccessTokenEqual(AuthResult o){
```

Comment:
I would suggest direcltly taking the accessToken as the input to this method

Reply:
Updated